src/index.html

```diff
@@ -1,14 +1,52 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>moyo_1</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="logo"
+        width="40" height="40">
+      </a>
+      <nav class="nav">
+        <ul class="nav_menu">
+          <li class="nav_list">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link"
+             data-qa="hover" >Laptops & Computers</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Gadgets</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Tablets</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Photo</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
   </body>
```

Comment:
```suggestion
    </header>
  </body>
```

Just remove empty line between header and body tags